ppr-ui/src/components/mhrRegistration/ReviewConfirm/HomeLocationReview.vue

```diff
@@ -223,6 +223,28 @@
               </p>
             </v-col>
           </v-row>
+
+          <div class="px-4">
+            <v-divider />
+          </div>
+
+          <!-- Land Details -->
+          <v-row no-gutters class="px-6 pt-6" >
+            <v-col cols="3" class="pt-1">
+              <h3>Land Details</h3>
```

Comment:
Is there anything else in Land Details? Looks empty.

Reply:
Was this part of the ticket then?

Reply:
Given that it was part of the design I believe so:

![image](https://github.com/bcgov/ppr/assets/77707952/ce932b21-321a-4646-be9b-b79124039e81)

Reply:
Oh I see, it sounds like a sub-header but looks like a label 😃 thanks for screenshot